testdata/baselines/reference/submoduleAccepted/conformance/jsDeclarationsFunctionPrototypeStatic.errors.txt.diff

```diff
@@ -0,0 +1,23 @@
+--- old.jsDeclarationsFunctionPrototypeStatic.errors.txt
++++ new.jsDeclarationsFunctionPrototypeStatic.errors.txt
+@@= skipped -0, +0 lines =@@
+-<no content>
++source.js(1,1): error TS2309: An export assignment cannot be used in a module with other exported elements.
```

Comment:
This seems preexisting; the reparsed code triggers some other errors about addtional exports.